lib/runners/swift.js

```diff
@@ -52,7 +52,7 @@ module.exports.run = function run(opts, cb) {
         args.push('/workspace/setup.swift');
       }
       fs.writeFileSync('/workspace/solution.swift', opts.solution);
-      fs.writeFileSync('/workspace/main.swift', opts.fixture);
+      fs.writeFileSync('/workspace/main.swift', opts.fixture.replace(/^\s*XCTMain\(/m, '_XCTMain('));
```

Comment:
Added whitespace flexibility, but no leading `_` allowed. There shouldn't be any content using `_XCTMain` yet right?